src/middlewares/checkJwt.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response, NextFunction } from 'express'
+import * as jwt from 'jsonwebtoken'
+import config from 'config'
+
+export function checkJwt(request: Request, response: Response, next: NextFunction) {
+    const token = <string>request.headers.authorization;
+    let jwtPayload;
```

Comment:
You should declare an interface with the contents of the jwtPayload to take advantage of TypeScript